Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/02-Navigation/04-NavigationStacks/02-04-NavigationStacks.tutorial

```diff
@@ -101,6 +101,9 @@
         This represents the actions that can happen inside the stack, such as pushing or popping
         an element off the stack, or an action happening inside a particular feature inside the
         stack.
+
+        > Tip: ``ComposableArchitecture/StackActionOf`` is a typealias to avoid having 
+        > to specify both `State` and `Action` (eg: `StackAction<ContactDetailFeature.State, ContactDetailFeature.Action>`)
```

Comment:
📓 I've added a comment to explain how it works without the typealias… not sure this is the best way to put it 🤔 
Feel free to suggest better words!